src/coreclr/jit/compiler.h

```diff
@@ -6246,11 +6246,11 @@ class Compiler
     PhaseStatus optOptimizeLayout(); // Optimize the BasicBlock layout of the method
     PhaseStatus optFindLoops();      // Finds loops and records them in the loop table
 
-    void optCloneLoops();
+    PhaseStatus optCloneLoops();
     void optCloneLoop(unsigned loopInd, LoopCloneContext* context);
     void optEnsureUniqueHead(unsigned loopInd, BasicBlock::weight_t ambientWeight);
-    void optUnrollLoops(); // Unrolls loops (needs to have cost info)
-    void optRemoveRedundantZeroInits();
+    PhaseStatus optUnrollLoops(); // Unrolls loops (needs to have cost info)
```

Comment:
I sure hope `ComponentActivator` warnings are unrelated to loop unrolling. Bad merge?

Reply:
Hmmm... definitely wasn't trying to touch this file. I rebased to origin since #52097 broke the build and was affecting my CI run